src/Commands/StartRoadRunnerCommand.php

```diff
@@ -85,13 +89,11 @@ public function handle(ServerProcessInspector $inspector, ServerStateFile $serve
             '-o', 'logs.output=stdout',
             '-o', 'logs.encoding=json',
             '--dotenv=""', 'serve',
```

Comment:
You can safely delete this CLI arg, by default RR doesn't use dotenv anymore in the latest version.